packages/graphql-elasticsearch-transformer/streaming-lambda/python_streaming_function.py

```diff
@@ -212,7 +215,11 @@ def _lambda_handler(event, context):
             es_actions.append(json.dumps(action))
             # Payload line
             es_actions.append(doc_json)
-
+            # migration step remove old key if it exists
+            if ('id' in doc_fields) and (event_name == 'MODIFY') :
```

Comment:
I am wondering if we should this route at all? This solves the problem for now but once every installation has this problem fixed, do we need this code? Would it become cruft?

Reply:
Yes that is correct we can add an environment variable into the lambda to run this piece and have that disabled at a later time when it is not needed?

Reply:
Yeah, I think this is okay to have here - On the top of my head, I don't see a cleaner and easier way to mitigate the re-indexing issue on the fly other than having it in the streaming function.